### PR TITLE
Update EigengrausGenerator.yml

### PR DESCRIPTION
**Pull request by WishingOnAWendy**
Change which upforgrabs tag is used

### CODE REVIEW

File: _data/projects/EigengrausGenerator.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ tags:
 - generation
 - tabletop
 upforgrabs:
-  name: good first issue
+  name: help wanted
   link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/EigengrausGenerator.yml]: To keep this in sync with the label change:

```suggestion
   link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/help%20wanted
```

**Review by shiftkey — CHANGES_REQUESTED** [review]

File: _data/projects/EigengrausGenerator.yml (hunk 1 above)
Comment (WishingOnAWendy) [reply _data/projects/EigengrausGenerator.yml]: Ah thanks, missed that

File: _data/projects/EigengrausGenerator.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ tags:
 - generation
 - tabletop
 upforgrabs:
-  name: good first issue
-  link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/good%20first%20issue
+  name: help wanted
+   link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/help%20wanted
```
Comment (shiftkey) [inline _data/projects/EigengrausGenerator.yml]: Ugh, I didn't notice this whitespace change I introduced. Apologies:

```suggestion
  link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/help%20wanted
```

**Review by shiftkey — APPROVED** [review]
Thanks @WishingOnAWendy!

**Pull request merged**